### PR TITLE
fix(#28): changed fakercloud url to cloudflare-ipfs.com

### LINKED ISSUE #2: Remove refs to old Marak repo
- [ ] Remove Faker Cloud Links
- [ ] Remove refs to `github.com/marak/faker.js
- [ ] Remove any references to non-MIT licensed content

### PR DESCRIPTION
**Pull request by prisis**
…ar endpoint

### CODE REVIEW

**Review by clarkerican — APPROVED** [review]

**Comment by prisis** [comment]
many thanks to @bchavez hope its fine if we use the same url for faker

File: lib/internet.js [hunk 1]
```diff
@@ -12,15 +12,12 @@ var Internet = function (faker) {
...
-      'https://cdn.fakercloud.com/avatars/' +
-      faker.random.arrayElement(faker.definitions.internet.avatar_uri)
-    );
+    return 'https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/' + faker.random.number(1249) + '.jpg';
   };
 
   self.avatar.schema = {
-    description: 'Generates a URL for an avatar.',
-    sampleResults: ['https://cdn.fakercloud.com/avatars/sydlawrence_128.jpg'],
+    "description": "Generates a URL for an avatar.",
```
Comment (Shinigami92) [inline lib/internet.js:19]: **suggestion:** Are these changes really be made by prettier? I assume not :thinking: 
Could you please run prettier for these two files? :slightly_smiling_face:

File: lib/internet.js [hunk 2]
```diff
@@ -12,15 +12,12 @@ var Internet = function (faker) {
    * @method faker.internet.avatar
    */
   self.avatar = function () {
-    return (
-      'https://cdn.fakercloud.com/avatars/' +
-      faker.random.arrayElement(faker.definitions.internet.avatar_uri)
-    );
+    return 'https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/' + faker.random.number(1249) + '.jpg';
```
Comment (Shinigami92) [inline lib/internet.js:15]: **question:** Could you explain `Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye`?

Comment (prisis) [reply lib/internet.js:15]: Yeah, this is the url where all the avatars a hosted, bchavez/Bogus#336

**Review by Shinigami92 — APPROVED** [review]

**Comment by Shinigami92** [comment]
I will bump the prio due to the current code is not working anymore

**Comment by bchavez** [comment]
Hi @prisis, I'm sorry for the late reply; currently, my workload at my job is high and I don't get out until late. :(

Yes, of course, everyone is welcome to use the `cloudflare-ipfs.com` URL from Bogus -- source code is here: https://github.com/bchavez/Bogus/blob/321341a137a7f46e09bbabff78c3215f6c6c6ffa/Source/Bogus/DataSets/Internet.cs#L37-L38

Seems this PR will use the correct URL. :+1: As mentioned earlier, anyone is free to help host and seed the avatar images on IPFS. The details on how to help seed the files can be found here: https://github.com/bchavez/Bogus/issues/336

With IPFS we can all help host static content in a decentralized way. The [CloudFlare IPFS (public service)](https://developers.cloudflare.com/distributed-web/ipfs-gateway) takes care of bridging web2 clients (HTTP browsers) with the IPFS network. Also, we have pinned the `Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye` Content ID (CID) folder with pinning services like Pinata Cloud (https://www.pinata.cloud/) to **_also_** help seed/pin the files on IPFS so they don't go away. (creating account is free)

The more people that help seed `Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye` the more resilient the URLs become.

There is almost no hosting bandwidth required to seed&host because CloudFlare's IPFS gateway heavily caches the static content files coming in from "the HTTP internet". Only one content seed is needed in case a file drops out of cache from CloudFlare.

For reference and a primer on IPFS can be found here:
* https://flyingzumwalt.gitbooks.io/decentralized-web-primer/content/

We have been using the `cloudflare-ipfs.com` URL for a long time and not a single problem :)

I hope this helps!
Brian Chavez

**Pull request merged**